hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyRequest.java

```diff
@@ -234,6 +239,19 @@ public void setup() throws Exception {
     when(ozoneManager.getOmSnapshotManager())
         .thenReturn(omSnapshotManager);
 
+
+    final boolean keyPathLockEnabled =
+        ozoneConfiguration.getBoolean(OZONE_OM_ENABLE_FILESYSTEM_PATHS,
+            OZONE_OM_ENABLE_FILESYSTEM_PATHS_DEFAULT);
+    final boolean fileSystemPathsEnable =
```

Comment:
Can we parameterize the tests based on the configuration flags passed: `ozone.om.key.path.lock.enabled` and `ozone.om.enable.filesystem.paths` to either use `OBSKeyPathLockStrategy` (finer `KEY_PREFIX_LOCK`) or `RegularBucketLockStrategy` (`BUCKET_LOCK`) pattern implementations depending on this [logic](https://github.com/apache/ozone/blob/master/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneLockProvider.java#L37-L52)?